httpstan/models.py

```diff
@@ -109,6 +127,23 @@ def _load_module(module_name: str, module_path: str):
     return module
 
 
+def _find_module(module_name: str, module_path: str):
```

Comment:
Could we use this, `_find_module` instead of `__load_module`? It would be nice to only maintain one of these.

Reply:
Yes, we could